compute_endpoint/globus_compute_endpoint/engines/high_throughput/interchange.py

```diff
@@ -356,10 +356,7 @@ def load_config(self):
             self.provider.channel.script_dir = os.path.join(
                 working_dir, "submit_scripts"
             )
-            self.provider.channel.makedirs(
-                self.provider.channel.script_dir, exist_ok=True
-            )
-            os.makedirs(self.provider.script_dir, exist_ok=True)
+            os.makedirs(self.provider.channel.script_dir, exist_ok=True)
```

Comment:
The following matches the previous functionality:
```suggestion
            os.makedirs(self.provider.channel.script_dir, mode=0o700, exist_ok=True)
            os.makedirs(self.provider.script_dir, exist_ok=True)
```